installer/mac/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#Check to see if version has been passed
+if [ -z "$1" ]
+  then
+    echo "No version tag passed"
+    exit 1
+fi
+
+#Get deploy script directory
+SCRIPTDIR=`dirname $0`
+#Build
+xcodebuild -configuration Release -scheme 'Space Evaders' -project $SCRIPTDIR/Space\ Evaders.xcodeproj
+#Move to src dir
+cd /Users/williampoynter/Documents/Personal/Coding/C++/Space\ Evaders/bin/Release
```

Comment:
Any chance you could pull this out into a config file so it isn't specific to your machine?